src/hotspot/share/opto/chaitin.cpp

```diff
@@ -1471,6 +1471,48 @@ static OptoReg::Name find_first_set(LRG& lrg, RegMask& mask) {
   return assigned;
 }
 
+OptoReg::Name PhaseChaitin::select_bias_lrg_color(LRG &lrg) {
+  uint bias_lrg1_idx = lrg._copy_bias;
```

Comment:
Can you enforce an invariant that `_copy_bias2 != 0` implies `_copy_bias != 0`?

Reply:
Bias live ranges are indipendent while marking and during enforcement.